package.json

```diff
@@ -86,6 +88,6 @@
     "lint-staged": "^10.4.0",
     "prettier": "^2.1.2",
     "react-app-rewired": "^2.1.6",
-    "typescript": "~3.7.2"
+    "typescript": "4.0.5"
```

Comment:
CRA v3 doesn't have support for Typescript v4.
In order to use TS v4, you need to use CRA v4, please undo this change